srcpkgs/python-numpy/template

```diff
@@ -15,6 +14,17 @@ distfiles="https://github.com/numpy/numpy/archive/v${version}.tar.gz"
 checksum=3c82a9b8616e3096a79a2af9c288d8ed4013a10fc7baf3eaf54655309734dadd
 alternatives="numpy:f2py:/usr/bin/f2py2"
 
+post_patch() {
+	case "${XBPS_TARGET_MACHINE}" in
+		armv5tel-musl)
+			cp "${XBPS_SRCPKGDIR}/python3-numpy/files/fenv-constants.h" \
```

Comment:
`$FILESDIR`

Reply:
No, `python3-numpy` is seperate to `python-numpy`.

Reply:
Could it be symlinked instead?

Reply:
i guess

Reply:
I was thinking more of symlinking the entire folder D:

Reply:
I don't really like that.. the `files/` folder should be independent of any one specific patch/fix that is applied.

Reply:
I think both of these options are bad. If somebody changes or removes the `python3-numpy` files (which becomes more likely the further the versions of the py2 and py3 packages diverge), the symlink or the cross-template copy will break. If this package needs this header, let it own a copy.

Reply:
I agree, I only changed it because @ericonr asked for it, but I can change it back.

Reply:
I've moved it back.